build/makelib/common.mk

```diff
@@ -75,7 +75,7 @@ endif
 
 # REAL_HOST_PLATFORM is used to determine the correct url to download the various binary tools from and it does not use
 # HOST_PLATFORM which is used to build the program.
-REAL_HOST_PLATFORM=$(shell go env GOHOSTOS)_$(GOHOSTARCH)
+REAL_HOST_PLATFORM := $(shell go env GOHOSTOS)_$(GOHOSTARCH)
```

Comment:
Why?

Reply:
by this, we can keep the `REAL_HOST_PLATFORM` variable as simply expanded variables, so by which it can be assigned only once.

Reply:
So by using `:=` we cannot override `REAL_HOST_PLATFORM` anymore?

Reply:
Yes.

Reply:
It can be overridden, but not if value that is been assigned changed later,

An example: 
1)  =
```
HELLO = world
HELLO_WORLD = $(HELLO) world!

# This echoes "world world!"
echo $(HELLO_WORLD)

HELLO = hello

# This echoes "hello world!"
echo $(HELLO_WORLD)
```
&&
2) :=
```
HELLO = world
HELLO_WORLD := $(HELLO) world!

# This echoes "world world!"
echo $(HELLO_WORLD)

HELLO = hello

# Still echoes "world world!"
echo $(HELLO_WORLD)

HELLO_WORLD := $(HELLO) world!

# This echoes "hello world!"
echo $(HELLO_WORLD)
```

Reply:
Ok, I'm not sure what improvement this brings...

Reply:
Currently, there is no effect using :=,
I thought `ifeq ($(GOARCH),amd64)` would be an improvement

I can go and close this anyway